Networking/Networking/Network/AlamofireNetwork.swift

```diff
@@ -19,7 +19,8 @@ public class AlamofireNetwork: Network {
     ///
     public required init(credentials: Credentials) {
         self.credentials = credentials
-        let configuration = URLSessionConfiguration.background(withIdentifier: "com.automattic.woocommerce.backgroundsession")
+        let uniqueID = UUID().uuidString
+        let configuration = URLSessionConfiguration.background(withIdentifier: "com.automattic.woocommerce.backgroundsession.\(uniqueID)")
```

Comment:
What do you think about adding here a small comment describing why we are using the uniqueID? I think it can be useful for the future

Reply:
yup! added in: https://github.com/woocommerce/woocommerce-ios/pull/2342/commits/5c012d0ab829219e7f5a08526b65fc74001a44de